mongodbatlas/cloud_provider_snapshot_backup_policies.go

```diff
@@ -76,6 +79,22 @@ type Export struct {
 	FrequencyType  string `json:"frequencyType,omitempty"`  // Frequency associated with the export policy.
 }
 
+// CopySetting is autogenerated from the json schema.
+type CopySetting struct {
+	CloudProvider     *string  `json:",omitempty"`
+	RegionName        *string  `json:",omitempty"`
+	ReplicationSpecID *string  `json:",omitempty"`
+	ShouldCopyOplogs  *bool    `json:",omitempty"`
+	Frequencies       []string `json:",omitempty"`
+}
+
+// DeleteCopiedBackup is autogenerated from the json schema.
+type DeleteCopiedBackup struct {
+	CloudProvider     *string `json:",omitempty"`
+	RegionName        *string `json:",omitempty"`
+	ReplicationSpecID *string `json:",omitempty"`
```

Comment:
can you please use the tag names for json here? specially since `ReplicationSpecID` is different from the expected `replicationSpecId`

Reply:
Tag names are added for json.